packages/runtime/src/modules/DeciderModule.ts

```diff
@@ -22,18 +75,261 @@ export class DeciderModule extends RuntimeModule {
     private async handleIncomingRequestStatusChanged(event: IncomingRequestStatusChangedEvent) {
         if (event.data.newStatus !== LocalRequestStatus.DecisionRequired) return;
 
-        if (event.data.request.content.items.some(flaggedAsManualDecisionRequired)) return await this.requireManualDecision(event);
+        const requestContent = event.data.request.content;
+        if (this.containsItem(requestContent, (item) => item["requireManualDecision"] === true)) {
+            return await this.requireManualDecision(event);
```

Comment:
That the setting for whether a request can be decided automatically is not made on the entire request seems kinda strange